app/_data/docs_nav_kic_2.7.x.yml

```
@@ -148,5 +150,5 @@ items:
         url: /references/prometheus
       - text: Feature Gates
         url: /references/feature-gates
-      - text: Gateway API Support
+      - text: Supported Gateway API Features
```

```suggestion
      - text: Gateway API Support
```
This section is getting revamped for https://github.com/Kong/kubernetes-ingress-controller/issues/3215 separately, so just leaving it as-is for now.